nginx-controller/controller/controller.go

```diff
@@ -467,7 +490,7 @@ func (lbc *LoadBalancerController) createIngress(ing *extensions.Ingress) nginx.
 			glog.Warningf("Error retrieving secret %v for Ingress %v: %v", secretName, ing.Name, err)
 			continue
 		}
-		ingEx.Secrets[secretName] = secret
+		ingEx.Secrets[fmt.Sprintf("%s-%s", ing.GetNamespace(), secretName)] = secret
```

Comment:
This creates a bug. The code in the `configurator` still uses the `secretName` as a key.

Better to solve the problem https://github.com/nginxinc/kubernetes-ingress/issues/89 in a separate pull request.